docs/source/docs/command-reference/context_free/transform.md

```diff
@@ -498,6 +499,29 @@ Examples:
     --id '2010_001705:train' --attr 'occluded'
   ```
 
+#### `astype_annotations`
+
+Enables the conversion of annotation types for the categories and individual items within a dataset.
+
+Based on a specified mapping, it transforms the annotation types, changing them to 'Label' if they are categorical,
+and to 'Caption' if they are of type string, float, or integer.
+
+Usage:
+```console
+astype_annotations [-h] [--mapping MAPPING]
+```
+
+Optional arguments:
+- `-h`, `--help` (flag) - Show this help message and exit
+- `--mapping` (str) - Annotations type in the form of: '<src>:<dst>' (repeatable)
+
+Examples:
+- Convert type of `title` and `rating` annotation
+  ```console
+  datum transform -t astype_annotations -- \
+    --mapping 'title:Caption,rating:int'
```

Comment:
I am worried that this kind of mapping makes misunderstanding. That is, users may not familiar with what Caption is. Is there any guideline how `int` can be decided?

Reply:
If rating is for classification, how about saying Label or Categorical?

Reply:
I totally agree with you. How about to use `text` and `label` to set annotation type? For example, `title:text,rating:label`.

Reply:
it looks good, because we have AnnotationType.label and text (by replacing caption in the future)